odl/deform/oit_reconstruction_in_operator.py

```diff
@@ -0,0 +1,762 @@
+# Copyright 2014-2016 The ODL development group
```

Comment:
I'm not quite sure what this file is meant to do, but it feels severely malplaced. The `Functional` stuff should not be here (it has its own branch) and all the manual tests should be moved somewhere else. Also, the dependence on `ddmatch` should not be in ODL.